apps/src/netsim/NetSimTable.js

```diff
@@ -217,8 +210,7 @@ NetSimTable.prototype.subscribe = function () {
  * local channel. Also clears the saved callback.
  */
 NetSimTable.prototype.unsubscribe = function () {
-  this.channel_.unsubscribe(this.tableName_, this.channelCallback_);
-  this.channelCallback = undefined;
+  this.channel_.unsubscribe(this.tableName_);
```

Comment:
oh right! I got lost in the hierarchy; too many things with `.unsubscribe` methods.